lib/APPReinforcementLearning/src/App.cpp

```diff
@@ -137,10 +188,89 @@ void App::loop()
  * Private Methods
  *****************************************************************************/
 
+void App::sendLineSensorsData() const
+{
+    ILineSensors&   lineSensors      = Board::getInstance().getLineSensors();
+    uint8_t         maxLineSensors   = lineSensors.getNumLineSensors();
+    const uint16_t* lineSensorValues = lineSensors.getSensorValues();
+    uint8_t         lineSensorIdx    = 0U;
+    LineSensorData  payload;
+
+    if (LINE_SENSOR_CHANNEL_DLC == maxLineSensors * sizeof(uint16_t))
```

Comment:
Use parenthesis on inside the comparison
```suggestion
    if (LINE_SENSOR_CHANNEL_DLC == (maxLineSensors * sizeof(uint16_t)))
```